pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -75,17 +75,14 @@
 from pytket.utils import prepare_circuit
 from pytket.utils.outcomearray import OutcomeArray
 from pytket.utils.results import KwargTypes
-from qiskit.primitives import (  # type: ignore
-    BitArray,
-    DataBin,
-    PrimitiveResult,
```

Comment:
Slightly concerned that I messed this file up when merging in `main`. Does it look okay to you @cqc-alec ?

Reply:
In fact, I think its fine... Any discrepancy with main should manifest as a merge coinflcit.

Reply:
No this is not right, looks like you've reverted the changes in #407 .

Reply:
Sorry, will fix.

Reply:
[c347b2a](https://github.com/CQCL/pytket-qiskit/pull/405/commits/c347b2a277cb8d637ad6179065fee3de09491899)